package.json

```diff
@@ -1,37 +1,38 @@
 {
   "name": "rythm.js",
-  "version": "2.0.7",
+  "version": "2.1.0",
   "description": "",
   "main": "rythm.js",
   "files": [
-    "rythm.js"
+    "rythm.js",
+    "rythm.min.js"
   ],
   "scripts": {
-    "start": "concurrently --kill-others  \"npm run serve\" \"rollup -c -w\"",
+    "start": "concurrently --kill-others  \"npm run serve\" \"npm run build -w\"",
     "serve": "gulp serve",
     "prepublish": "npm run build",
     "test": "echo \"No test specified\" && exit 0",
-    "build": "rollup -c"
+    "build": "node build.js"
   },
   "author": "Benjamin Plouzennec <plouzennec.benjamin@gmail.com>",
   "license": "GNU GPL",
-  "dependencies": {
```

Comment:
@Okazari Just tagging you 'cause you keep missing that I fixed this in the second commit of my PR ;)

Reply:
Right i'm blind :scream: